docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3"
+services:
+  consul:
+    container_name: cert-manager.devstack.consul
+    image: consul:1.2.2
+    environment: 
+      CONSUL_BIND_INTERFACE: eth0
+      PIPENV_VENV_IN_PROJECT: "true"
+      LC_ALL: "C.UTF-8"
+      LANG: "C.UTF-8"
+    tty: true
+    ports:
+      - "8500:8500"
+    volumes: 
+      - "$PWD:/opt/"    
+    networks:
+      cert-manager_default:
+        ipv4_address: 192.168.176.3
+  test:
+    container_name:  cert-manager.devstack.test
+    image: python:3.6
+    environment: 
+      CONSUL_HTTP_ADDR: 192.168.176.3:8500
```

Comment:
The custom networks can be removed, and this value replaced with `cert-manager.devstack.consul:8500` (relying on docker's pseudo dns system).